.gitignore

```diff
@@ -1,5 +1,7 @@
 # See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
 
+.env
+
```

Comment:
Unusually we commit .env for this project so this ignore is wrong. I don't think we even build without it being persent.

I'll have a think about this as I'm not sure a mix of approaches across projects is a good idea.

Reply:
The Vite way seems to be to commit defaults in .env and use .env.local for uncommitted changes. So this is fine and I might update the approach elsewhere.

Reply:
The ignore however is wrong